### PR TITLE
Updates to lambda transofrmation to make it Windows compatible

### PR DESCRIPTION
**Pull request by jonathan-owens**
Move from using a lambda in transforms.Compose to using transforms.Lambda, so that the transforms `list` can be pickled on Windows and used with multiple `DataLoader` workers.

This alleviates errors of this nature:

`C:/w/1/s/tmp_conda_3.7_055457/conda/conda-bld/pytorch_1565416617654/work/aten/src/THC/THCTensorScatterGather.cu:100: block: [8,0,0], thread: [447,0,0] Assertion `indexValue >= 0 && indexValue < src.sizes[dim]` failed.`

See problem discussion [here](https://github.com/belskikh/kekas/issues/26) and see [here](https://discuss.pytorch.org/t/cant-pickle-local-object-dataloader-init-locals-lambda/31857/10) for discussion of the implemented solution.

### CODE REVIEW

**Comment by bshall** [comment]
Hi @jonathan-owens,

Thanks for the PR. Everything works as expected on my side so happy to merge :thumbsup:
